libraries/adaptive-expressions/tests/badExpression.test.js

```diff
@@ -420,7 +420,6 @@ const badExpressions = [
             ['where(items, item, item2, item3)', 'should have three parameters'],
             ['where(items, add(1), item)', 'Second paramter of where is not an identifier'],
             ['where(items, 1, item)', 'Second paramter error'],
-            ['where(items, x, sum(x))', 'third paramter error'],
```

Comment:
Why is this being removed?

Reply:
Currently SDK would treat the "error item" in `Where` iteration as a "false" result and drop it to avoid too much errors.